src/components/TextInput/baseTextInputPropTypes.js

```diff
@@ -19,6 +19,9 @@ const propTypes = {
     /** Error text to display */
     errorText: PropTypes.string,
 
+    /** Icon to display in right side of text input */
+    icon: PropTypes.func,
```

Comment:
If the default value is `null`, should we allow that in the prop type here?

Reply:
@Beamanator No, no need since the icon is not required, having a default of `undefined` or `null` is okay.

Reply:
aah ok great! I wasn't sure about `null` being ok in this situation :D 👍